src/master-playlist-controller.js

```diff
@@ -711,454 +625,15 @@ export class MasterPlaylistController extends videojs.EventTarget {
-      kind = 'main-desc';
-    }
-
-    return kind;
-  }
-  /**
-   * fill our internal list of Subtitle Tracks with data from
-   * the master playlist or use a default
-   *
-   * @private
-   */
-  fillSubtitleTracks_() {
-    let master = this.master();
-    let mediaGroups = master.mediaGroups || {};
-
-    for (let mediaGroup in mediaGroups.SUBTITLES) {
-      if (!this.subtitleGroups_.groups[mediaGroup]) {
-        this.subtitleGroups_.groups[mediaGroup] = [];
-      }
-
-      for (let label in mediaGroups.SUBTITLES[mediaGroup]) {
-        let properties = mediaGroups.SUBTITLES[mediaGroup][label];
-
-        if (!properties.forced) {
-          this.subtitleGroups_.groups[mediaGroup].push(
-            videojs.mergeOptions({ id: label }, properties));
-
-          if (typeof this.subtitleGroups_.tracks[label] === 'undefined') {
-            let track = this.tech_.addRemoteTextTrack({
-              id: label,
-              kind: 'subtitles',
-              enabled: false,
-              language: properties.language,
-              label
-            }, false).track;
-
-            this.subtitleGroups_.tracks[label] = track;
-          }
-        }
-      }
-    }
-
-    // Do not enable a default subtitle track. Wait for user interaction instead.
-  }
-
-  /**
-   * fill our internal list of Captions Tracks with data from
-   * the master playlist or use a default
-   *
-   * @private
-   */
-  fillClosedCaptionTracks_() {
-    let master = this.master();
-    let mediaGroups = master.mediaGroups || {};
-
-    for (let mediaGroup in mediaGroups['CLOSED-CAPTIONS']) {
-      if (!this.closedCaptionGroups_.groups[mediaGroup]) {
-        this.closedCaptionGroups_.groups[mediaGroup] = [];
-      }
-
-      for (let label in mediaGroups['CLOSED-CAPTIONS'][mediaGroup]) {
-        let properties = mediaGroups['CLOSED-CAPTIONS'][mediaGroup][label];
-
-        // We only support CEA608 captions for now, so ignore anything that
-        // doesn't use a CCx INSTREAM-ID
-        if (!properties.instreamId.match(/CC\d/)) {
-          continue;
-        }
-
-        this.closedCaptionGroups_.groups[mediaGroup].push(
-          videojs.mergeOptions({ id: label }, properties));
-
-        if (typeof this.closedCaptionGroups_.tracks[label] === 'undefined') {
-          let track = this.tech_.addRemoteTextTrack({
-            id: properties.instreamId,
-            kind: 'captions',
-            enabled: false,
-            language: properties.language,
-            label
-          }, false).track;
-
-          this.closedCaptionGroups_.tracks[label] = track;
-        }
-      }
-    }
-
-  }
-
-  /**
    * Call load on our SegmentLoaders
    */
   load() {
     this.mainSegmentLoader_.load();
-    if (this.audioPlaylistLoader_) {
+    if (this.mediaGroups_.AUDIO.activePlaylistLoader) {
       this.audioSegmentLoader_.load();
     }
-    if (this.subtitlePlaylistLoader_) {
```

Comment:
I'm not sure what you are asking. `load` is still called on the `subtitleSegmentLoader_` it's just that the check for the playlist loader was changed to a different reference

Reply:
Maybe there is confusion because git diff is matching the load call with one much lower from another function that was removed? (line 1159 for this file, line 636 in the new file)?

Reply:
Yup, it was the diff 👍